WordPress/UITestsFoundation/WireMock.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+public class WireMock {
+
+    public static func URL() -> Foundation.URL {
+        let host = ProcessInfo().environment["WIREMOCK_HOST"] ?? "localhost"
+        let port = ProcessInfo().environment["WIREMOCK_PORT"] ?? "8282"
```

Comment:
The previous implementation of this class would read the host and port values from the `Bundle`, which picked them up from the `Info.plist`, which in turn read them from user defined build settings.

Once I moved the class to a different target than the one were those user defined build settings were set, the setup broke because the `Info.plist` of the new UITestsFoundation target doesn't have those values.

But, since the were hardcoded in the project file anyways and there was no logic in our automation that passed custom `xcargs` values to override them, did they really need to be read from there anymore?

This solution allows the class to live in this target without any change to its external API. By using the launch environment variables, we can also retain the ability to override the settings.

Unless I'm missing some context on the previous implementation, I think in this new iteration of the architecture this approach is a good fit.